src/client/src/app/lib/services/timeService.ts

```diff
@@ -343,3 +343,14 @@ export const addHoursAndMinutesToUkDatetime = (
   const newHour = addToUkDatetime(ukDatetime, hours, 'hour', dateTimeFormat);
   return addToUkDatetime(newHour, minutes, 'minute', dateTimeFormat);
 };
+
+export const emptyWeekSummary: WeekSummary = {
+  daySummaries: [],
+  maximumCapacity: 0,
+  bookedAppointments: 0,
+  orphanedAppointments: 0,
+  cancelledAppointments: 0,
+  remainingCapacity: 0,
+  startDate: dayjs.utc(),
+  endDate: dayjs.utc(),
+};
```

Comment:
It feels a bit odd this living in the `timeService`, I'd have expected it in `@types`. Is it only here because it's using the `dayjs` object? If so, could it accept these dates as ISO strings or something instead?

Furthermore, is there a reason you're using utc here? I thought we outlawed uses of utc in favour of UK time?